### PR TITLE
pcg-c rng library

### PR DESCRIPTION
**Pull request by listx**
Hello, this is my first contribution to nixpkgs. I just copy-pasted-modded a lot of thing and tested out a pet project and it seems to work fine. Please critique if necessary, thanks.

### CODE REVIEW

**Comment by listx** [comment]
Oops, I forgot to add the patch file.

**Comment by listx** [comment]
Rebased. It should build fine now...

File: pkgs/development/libraries/pcg-c/default.nix [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchzip }:
+
+with stdenv.lib;
+
+stdenv.mkDerivation rec {
+  version = "0.94";
+  name = "pcg-c";
```
Comment (pSub) [inline pkgs/development/libraries/pcg-c/default.nix]: Please mention the version in the name. e.g.
```name = "pcg-c-${version}"```

**Comment by pSub** [comment]
Looks good otherwise.

**Comment by listx** [comment]
Thank you, I've added the version to the name.

**Comment by jagajaga** [comment]
Better to squash commits and mention about yourself as a maintainer in a commit msg.

**Comment by jagajaga** [comment]
Everything else is ok.

**Comment by listx** [comment]
@jagajaga OK, I've squashed it into 1 commit.

**Pull request merged**